src/view/maintoolbar.cpp

```diff
@@ -25,6 +25,7 @@ MainToolBar::MainToolBar(lib::spt::api &spotify, lib::settings &settings,
 	// Search
 	search = addAction(Icon::get("edit-find"), "Search");
 	search->setCheckable(true);
+	search->setShortcut(QKeySequence("Ctrl+f"));
```

Comment:
Actually, now that I think of it, there's already an enum for a find shortcut
```suggestion
	search->setShortcut(QKeySequence::Find);
```

Reply:
That's interesting, thank you! I've modified it.